frontend/packages/process-editor/src/components/ConfigPanel/ConfigEndEvent/ConfigEndEvent.tsx

```diff
@@ -1,19 +1,20 @@
 import React from 'react';
 import { StudioLabelAsParagraph, StudioSectionHeader } from '@studio/components';
-import { Link, Paragraph, Accordion } from '@digdir/design-system-react';
+import { Accordion, Link, Paragraph } from '@digdir/design-system-react';
 import { useTranslation } from 'react-i18next';
 import classes from './ConfigEndEvent.module.css';
 import { ConfigIcon } from '../ConfigContent/ConfigIcon';
 import { CustomReceiptContent } from './CustomReceiptContent';
 import { altinnDocsUrl } from 'app-shared/ext-urls';
+import { BpmnTypeEnum } from '@altinn/process-editor/enum/BpmnTypeEnum';
```

Comment:
Should avoid absolute imports until we have decided 100% on how we do this within the team 😄